genome_preparation.nf

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env nextflow
+
+def helpMessage() {
+    log.info"""
+    Index the provided genome with bowtie and samtools
+    Usage:
+            --fasta                     Fasta file containing the reference genome.
+            --outdir                    Directory where the index will be deployed.
+    """
+}
+
+// Show help message
+if (params.help) exit 0, helpMessage()
+
+params.fasta  = "$baseDir/resources/genome.fa"
+params.outdir = "$baseDir/resources/"
+
+
```

Comment:
```suggestion
// The basename variable contains the name and extension of the input file, e. g. "genome.fa"
```

Standard definition for the basename is "file name without the extension", so I think we should clarify what we mean by it here

Reply:
The diff makes it hard to see, but this suggested comment is right before the definition of "basename" in the file